nodebuilder/node.go

```diff
@@ -149,6 +149,7 @@ func (n *Node) Stop(ctx context.Context) error {
 func newNode(opts ...fx.Option) (*Node, error) {
 	node := new(Node)
 	app := fx.New(
+		fx.NopLogger,
```

Comment:
I'm a bit confused. My local main has `fx.NopLogger`, but our remote main has not. I understand why we need id, but why was it removed?

Reply:
It was remove by accident

Reply:
Can extract into a separate PR if necessary